prog/vm/github_runner.rb

```diff
@@ -138,13 +129,17 @@ def before_run
       github_runner.update(runner_id: response[:runner][:id], ready_at: Time.now)
       # ./env.sh sets some variables for runner to run properly
       vm.sshable.cmd("./env.sh")
-      vm.sshable.cmd("common/bin/daemonizer 'sudo -u runner /home/runner/run.sh --jitconfig #{response[:encoded_jit_config].shellescape}' runner-script")
+
+      dir = "/home/runner"
+      command = "sudo -u runner #{dir}/run.sh --jitconfig #{response[:encoded_jit_config].shellescape}"
```

Comment:
You know, we might not need prepending #{dir} anymore. Initially I put that to daemonizer because we didn't pass --working-directory. Now we are passing that, so prepending is probably unnecessary. I didn't test it though...

Reply:
Good catch. It's working without dir